### PR TITLE
document spending-company-money philosophy and allowable expenses

### PR DESCRIPTION
**Pull request by sqs**
A teammate just asked:

> Q: over past few months I have been having to download a lot of data for random stuff (load tests for search perf, downloading lots of repos onto dev machine for testing, etc) and it's put me over my 1TB data cap with my ISP a few times which is just kinda a PITA -- I finally just bit the bullet and got an unlimited plan for $50/mo extra, but this got me thinking:
>
> Do we currently have (or have we thought about having) some sort of reimbursement for remote folks internet plans? no biggie either way just curious

I replied:

> Seems like we should have a good plan here. I will start a stub PR on the handbook and assign it to @mercadon and @dadlerj to take it from here. I see GitLab reimburses internet connection subscriptions (https://about.gitlab.com/handbook/spending-company-money/).

At a minimum, use this PR to push a document to answer the teammate's question above. Feel free to write more, too, to answer other common questions and make us an aspirational team per our 2019Q4 OKR.

---

Also should answer the following questions:

- https://sourcegraph.slack.com/archives/C07KZF47K/p1573509369461400?thread_ts=1573509288.460900&cid=C07KZF47K
- https://sourcegraph.slack.com/archives/C07KZF47K/p1573631427398000

### CODE REVIEW

File: handbook/people-ops/spending_company_money.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Spending company money
```
Comment (sqs) [inline handbook/people-ops/spending_company_money.md]: You can rename this doc, I just used/copied what GitLab used at https://about.gitlab.com/handbook/spending-company-money/.

**Comment by nicksnyder** [comment]
I would love to have all our benefits documented here in the handbook (and we can link to it from the careers repo).

For internet specifically, I think we should do the same think we do for mobile phones (i.e. just reimburse everyone automatically at a fixed rate) because this basically applies to everyone.

**Comment by sqs** [comment]
@mercadon friendly ping on this

**Comment by mercadon** [comment]
Can now be found here: https://about.sourcegraph.com/handbook/people-ops/spending-company-money

**Pull request closed without merging**